src/sources/notification_permissions.test.ts

```diff
@@ -1,11 +1,12 @@
-import { getBrowserMajorVersion, isHeadlessChrome, isMobile, isWebKit } from '../../tests/utils'
+import { getBrowserMajorVersion, getBrowserVersion, isHeadlessChrome, isMobile, isWebKit } from '../../tests/utils'
 import { BotdError } from '../types'
 import getNotificationPermissions from './notification_permissions'
 
 describe('Sources', () => {
   describe('notificaionPermissions', () => {
     it('returns an expected value or throws', async () => {
-      if (isWebKit()) {
+      const { major, minor } = getBrowserVersion() ?? { major: 0, minor: 0 }
```

Comment:
This test is broken in default branch as well, I fix it here for simplicity

Reply:
cc @bayotop because you've added other edge cases to this test, maybe you're interested in the latest Safari changes (16.5 is the latest on browserstack, but perhaps it changed earlier in 16 version, I don't have other minor versions to check)